src/compiler/checker.ts

```diff
@@ -11027,6 +11027,10 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         }
 
         if (isParameter(declaration)) {
+            if (!declaration.symbol) {

```

Comment:
I decided to include this here because it's just always possible to call `checker.getTypeAtLocation` or `checker.getTypeAtLocation` and those shouldn't crash or do something overly goofy - so the safest bet is to actually be on the safe side of things here + fine-tune existing call sites to those 2 if it's absolutely necessary to avoid getting `errorType` back when it's obvious that it will be returned back